### PR TITLE
Bug/ens

### PR DESCRIPTION
**Pull request by gesseekur**
### Bug

- [x] Updated CHANGELOG.md
- [X] Add PR label

### CODE REVIEW

File: src/dapps/BuySubdomain/BuySubdomain.vue [hunk 1]
```diff
@@ -119,7 +122,12 @@ export default {
   },
   methods: {
     debounceInput: web3.utils._.debounce(function(e) {
-      this.domainName = normalise(e.target.value);
+      if (e.target.value.indexOf(' ') >= 0) {
```
Comment (kvhnuke) [inline src/dapps/BuySubdomain/BuySubdomain.vue]: it cannot also have `.` and other invalid characters also `normalise` can fail, may be put normalise inside try catch?

File: src/dapps/ManageENS/ABI/registrarAbi.js [hunk 2]
```diff
@@ -23,30 +23,6 @@ export default [
     payable: false,
```
Comment (kvhnuke) [inline src/dapps/ManageENS/ABI/registrarAbi.js]: I think we can safely delete this whole file? are we referencing this anywhere?

File: src/dapps/ManageENS/ManageENS.vue [hunk 3]
```diff
@@ -566,37 +514,6 @@ export default {
       }
     },
     processPermanentRegistrar() {},
```
Comment (kvhnuke) [inline src/dapps/ManageENS/ManageENS.vue]: empty `function`

File: src/dapps/ManageENS/ABI/registrarAbi.js (hunk 2 above)
Comment (gesseekur) [reply src/dapps/ManageENS/ABI/registrarAbi.js]: @kvhnuke It does look like its been referenced on ManageENS.vue.341

File: src/dapps/ManageENS/ManageENS.vue (hunk 3 above)
Comment (gesseekur) [reply src/dapps/ManageENS/ManageENS.vue]: removed!

File: src/dapps/BuySubdomain/BuySubdomain.vue (hunk 1 above)
Comment (gesseekur) [reply src/dapps/BuySubdomain/BuySubdomain.vue]: changed 👍

**Pull request merged**